meson_options.txt

```diff
@@ -472,6 +472,8 @@ option('fuzz-tests', type : 'boolean', value : false,
        description : 'run the fuzzer regression tests by default (with sanitizers)')
 option('install-tests', type : 'boolean', value : false,
        description : 'install test executables')
+option('unittestsdir', type : 'string', value : '',
```

Comment:
we shouldn't need an option for this, right?

Reply:
oops yeah a leftover... good catch.